packages/web/localizations/pl.json

```diff
@@ -501,5 +501,24 @@
   "seo.pool.title": "Basen #",
   "seo.pool.description": "Dodaj swoje aktywa do tej puli, aby odblokować RRSO. Im dłuższy okres rozłączenia, tym więcej zarabiasz na Osmosis - wiodącej zdecentralizowanej giełdzie Cosmos",
   "seo.pools.title": "baseny osmotyczne",
-  "seo.pools.description": "Odkryj pule płynności na Osmosis - wiodącej zdecentralizowanej giełdzie Cosmos"
+  "seo.pools.description": "Odkryj pule płynności na Osmosis - wiodącej zdecentralizowanej giełdzie Cosmos",
+  "stake.title": "Stawka",
+  "stake.available": "Dostępny",
+  "stake.HALF": "POŁOWA",
+  "stake.MAX": "MAKS",
+  "stake.validatorHeader": "Ekipa walidatorów",
```

Comment:
Looks like inlang still hasn't fixed and/or updated our version to add as structured JSON. But this will work for now and we can regenerate it later when they fix it